common/customproperties.go

```diff
@@ -19,9 +19,6 @@ type CustomProperties struct {
 	x *custom_properties.Properties
 }
 
-// CustomProperty contains document specific property
-type CustomProperty *custom_properties.CT_Property
```

Comment:
Cannot remove exported symbols without major upgrade.

Reply:
This caused an error in the playground (and anyway looks senseless)